.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: python
 python:
+  - "3.6"
+  - "3.7"
```

Comment:
I don't see the need for 3.7, since we don't intend to ever run that in production. If something worked on 3.6 & 3.8, we wouldn't want to fix 3.7. (A bit hypothetical, but basically a vote for 2 versions: current prod & next prod)